draft-ietf-dhc-addr-notification.md

```diff
@@ -322,22 +322,24 @@ If an ADDR-REG-REPLY message is received for the address being registered, the c
 
 ## Registration Expiry and Refresh
 
-The client MUST refresh registrations to ensure that the server is always aware of which addresses are still valid. The client SHOULD perform refreshes as described below. Each refresh is scheduled for AddrRegRefresh seconds in the future, where AddrRegRefresh is min(4 hours, 80% of the address's current Valid Lifetime). Refreshes SHOULD be jittered by +/- 10% to avoid synchronization causing a large number of registration messages from different clients at the same time.
+The client MUST refresh registrations to ensure that the server is always aware of which addresses are still valid. The client SHOULD perform refreshes as described below.
 
-Whenever the client creates an address or receives a PIO which changes the Valid Lifetime of an existing address by more than 1%, then:
+We define a function AddrRegRefreshInterval(address) as min(4 hours, 80% of the address's current Valid Lifetime). When calculating this value, the client SHOULD apply a jitter of +/- 10% to avoid synchronization causing a large number of registration messages from different clients at the same time.
 
-1. If no refresh is currently scheduled, it MUST register immediately and schedule a refresh.
-1. If a refresh is currently scheduled, it MUST reschedule the existing refresh if this would result in the refresh being sooner than currently scheduled.
+Whenever the client registers or refreshes an address, it calculates a NextAddrRegRefreshTime for that address as AddrRegRefreshInterval seconds in the future, but does not schedule any refreshes.
```

Comment:
...any furture/subsequent refreshes?

Reply:
Well, something that is scheduled generally *has* to be in the future... :-)